spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/core/mapping/Neo4jMappingContext.java

```diff
@@ -83,13 +85,18 @@ public Neo4jMappingContext() {
 	public Neo4jMappingContext(Neo4jConversions neo4jConversions) {
 
 		super.setSimpleTypeHolder(Neo4jSimpleTypes.HOLDER);
+		this.neo4jConversions = neo4jConversions;
 		this.converter = new DefaultNeo4jConverter(neo4jConversions, nodeDescriptionStore);
 	}
 
 	public Neo4jConverter getConverter() {
 		return converter;
 	}
 
+	boolean hasRegisteredConverter(Class<?> targetType) {
```

Comment:
```suggestion
	boolean hasCustomWriteTarget(Class<?> targetType) {
```

Can we have delegate method not renaming target when it's not necessary, please?